packages/calcite-components/src/components/list/list.e2e.ts

```diff
@@ -339,4 +339,87 @@ describe("calcite-list", () => {
     expect(await listItemOne.getProperty("selected")).toBe(false);
     expect(await list.getProperty("selectedItems")).toHaveLength(0);
   });
+
+  describe.only("keyboard navigation", () => {
+    async function evaluateActiveElement(page: E2EPage, activeElementSelector: string): Promise<void> {
+      expect(await page.evaluate((selector) => document.activeElement?.matches(selector), activeElementSelector)).toBe(
+        true
+      );
+    }
+
+    it("should navigate via ArrowUp, ArrowDown, Home, and End", async () => {
+      const page = await newE2EPage({
```

Comment:
For consistency, can you create the page and then populate it with `setContent`?